src/preact/base-element.js

```diff
@@ -212,32 +213,32 @@ export class PreactBaseElement extends AMP.BaseElement {
   /** @override @nocollapse */
   static requiresShadowDom() {
     // eslint-disable-next-line local/no-static-this
-    return this['usesShadowDom'];
+    return this['usesShadowDom']; // eslint-disable-line local/restrict-this-access
```

Comment:
These are defined statically on this class, so they're safe to add as exceptions.